docs/specs/legacy.yml

```diff
@@ -1,4 +1,5 @@
 # Build output as page json
+noContinue: true
```

Comment:
I don't quite following the _"cannot use the cache info in spec test"_ argument, the second stage presumably don't need cache info?